tests/ZeroMQMessageProducerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Prooph\ServiceBusTest;
```

Comment:
I thought that too but take a look at this one; https://github.com/prooph/psb-http-producer/blob/master/tests/HttpMessageProducerTest.php